modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main_view/props_transformer/ViewVersionHistoryFDSPropsTransformer.ts

```diff
@@ -51,21 +54,9 @@ export default function ViewVersionHistoryFDSPropsTransformer({
 			itemData,
 			loadData,
 		}: {
-			action: {data: {id: string; successMessage: string}; href?: string};
+			action: any;
 			event: Event;
-			itemData: {
-				actions: {
-					copy: {href: string; method: string};
-					delete: {href: string; method: string};
-					expire: {href: string; method: string};
-					restore: {href: string; method: string};
-				};
-				systemProperties: {
-					version: {
-						number: number;
-					};
-				};
-			};
+			itemData: any;
```

Comment:
We should avoid using type `any` as much as possible, can't we type is as needed.

Reply:
Yes, no problem. I am going to change it.